pkgs/by-name/co/cortex-command-community-project/package.nix

```
@@ -0,0 +1,79 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  flac,
+  lib,
+  libgcc,
+  libGL,
+  libpng,
+  libpulseaudio,
+  lz4,
+  luajit,
+  meson,
+  minizip,
+  ninja,
+  nix-update-script,
+  pkg-config,
+  SDL2,
+  SDL2_image,
+  tbb,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cortex-command-community-project";
+  version = "6.2.2";
+
+  src = fetchFromGitHub {
+    owner = "cortex-command-community";
+    repo = "Cortex-Command-Community-Project";
+    rev = "v${version}";
+    sha256 = "sha256-srbV6Nh+ecyV0dkY835vhzpMSzmnvANym453L72cmGI=";
```

```suggestion
    hash = "sha256-srbV6Nh+ecyV0dkY835vhzpMSzmnvANym453L72cmGI=";
```
The `sha256` [hash attribute](https://wiki.nixos.org/wiki/Nix_Hash) in fixed-output derivations has been replaced by the new `hash` attribute, which encourages the use of [SRI hashes](https://www.w3.org/TR/SRI/).